### PR TITLE
[Jira v2] Added 3 new commands

### PR DESCRIPTION
**Pull request by ennozdd**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added 3 new commands:
    *   jira-get-organizations
    *   jira-add-user-to-project
    *   jira-get-project-role

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Jira/Integrations/JiraV2/JiraV2.py [hunk 1]
```diff
@@ -723,6 +730,19 @@ def get_issue(issue_id, headers=None, expand_links=False, is_update=False, get_a
     return human_readable, outputs, contents
 
 
+def get_project_role_command(project_key, role_name):
+    url = f'rest/api/latest/project/{project_key}/role'
```
Comment (thefrieddan1) [inline Packs/Jira/Integrations/JiraV2/JiraV2.py:734]: Please extract lines 734 to 736 to a seperate method.

File: Packs/Jira/Integrations/JiraV2/JiraV2.py [hunk 2]
```diff
@@ -723,6 +730,19 @@ def get_issue(issue_id, headers=None, expand_links=False, is_update=False, get_a
...
+    j_res = jira_req('GET', url, resp_type='json')
+    url = j_res.get(role_name)
+    if not url:
+        return_error(f"Role '{role_name}' not found")
+
+    role_id = url.split('/')[-1]
```
Comment (thefrieddan1) [inline Packs/Jira/Integrations/JiraV2/JiraV2.py:740]: please extract line 740 to 742 to a seperate method.

**Review by thefrieddan1 — COMMENTED** [review]
Hi @ennozdd 
Thank you for your contribution.
Please create a unit test for each command added.

Thanks,

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml [hunk 3]
```diff
@@ -538,6 +541,21 @@ script:
   - description: Available from Cortex XSOAR version 6.1.0. This command queries for incidents that were modified since the last update. This method is only used for debugging.
     name: get-modified-remote-data
     arguments: []
+  - arguments: []
+    description: Gets the information related to the role, including users assigned.
```
Comment (ShirleyDenkberg) [inline Packs/Jira/Integrations/JiraV2/JiraV2.yml:545]:
```suggestion
    description: Gets the information related to the role, including assigned users.
```

File: Packs/Jira/Integrations/JiraV2/README.md [hunk 4]
```diff
@@ -1050,3 +1050,57 @@ Add a new custom field and add it to the incident type&#39;s layout:
...
+| project_key | Project key. | Optional | 
+| role_name | Role name to assign the user. | Optional | 
+
+#### Context Output
+
+There is no context output for this command.
+### jira-get-project-role
+
+***
+Gets the information related to the role, including users assigned.
```
Comment (ShirleyDenkberg) [inline Packs/Jira/Integrations/JiraV2/README.md:1076]:
```suggestion
Gets the information related to the role, including assigned users.
```

File: Packs/Jira/ReleaseNotes/3_0_3.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Atlassian Jira v2
+
+- Added the following new commands:
+    *   jira-get-organizations
```
Comment (ShirleyDenkberg) [inline Packs/Jira/ReleaseNotes/3_0_3.md:7]:
```suggestion
    *   ***jira-get-organizations***
```

File: Packs/Jira/ReleaseNotes/3_0_3.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+    *   jira-add-user-to-project
```
Comment (ShirleyDenkberg) [inline Packs/Jira/ReleaseNotes/3_0_3.md:8]:
```suggestion
    *   ***jira-add-user-to-project***
```

File: Packs/Jira/ReleaseNotes/3_0_3.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+    *   jira-get-project-role
```
Comment (ShirleyDenkberg) [inline Packs/Jira/ReleaseNotes/3_0_3.md:9]:
```suggestion
    *   ***jira-get-project-role***
```

**Comment by ShirleyDenkberg** [comment]
@thefrieddan1 Doc review completed.

**Comment by ShirleyDenkberg** [comment]
@ennozdd I already reviewed this PR

**Review by thefrieddan1 — APPROVED** [review]

**Pull request merged**